lib/driver.js

```diff
@@ -34,7 +34,8 @@ const DEFAULT_SETTINGS = {
   shouldUseCompactResponses: true,
   elementResponseAttributes: "type,label",
   mjpegServerScreenshotQuality: 25, // https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Utilities/FBConfiguration.m#L29
-  mjpegServerFramerate: 10 // https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Utilities/FBConfiguration.m#L30
+  mjpegServerFramerate: 10, // https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Utilities/FBConfiguration.m#L30
+  screenshotQuality: 1 // https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Utilities/FBConfiguration.m#L31
```

Comment:
these are probably not needed as well

Reply:
Currently, the key/value checks available settings using here. We can see below error if we have no value here.

So, I would like to create another PR to change the method.

```
[XCUITest] Setting the quality of phone screenshot: '2'
[BaseDriver] Didn't know about setting 'screenshotQuality'. Are you sure you spelled it correctly? Proceeding anyway. Valid settings: imageMatchThreshold,fixImageFindScreenshotDims,fixImageTemplateSize,checkForImageElementStaleness,autoUpdateImageElementPosition,imageElementTapStrategy,nativeWebTap,useJSONSource,shouldUseCompactResponses,elementResponseAttributes,mjpegServerScreenshotQuality,mjpegServerFramerate
```

Reply:
:note: https://github.com/appium/appium-base-driver/blob/f510db0257110a755c9f2976bf7580abc9402bc9/lib/basedriver/device-settings.js#L44

Reply:
Then it would be better to remove line numbers from comments, since these might change in the future